src/main/java/edu/harvard/iq/dataverse/settings/JvmSettings.java

```diff
@@ -150,6 +150,13 @@ public enum JvmSettings {
     SCOPE_NETCDF(PREFIX, "netcdf"),
     GEO_EXTRACT_S3_DIRECT_UPLOAD(SCOPE_NETCDF, "geo-extract-s3-direct-upload"),
 
+    // BAGIT SETTINGS
+    SCOPE_BAGIT(PREFIX, "bagit"),
```

Comment:
We already have a Brand name for the repository and associated email address. Should those be used here instead of new settings? If not, I don't know that these should be in a bagit scope - they probably could/should go in other exports, be available via API, etc. as generally usable info.

Reply:
I had that thought... I'm ok with reusing existing settings.

Should "mailing address" also be a more global setting? Not BagIt-specific, I mean?

Reply:
Yeah - not sure how settings will be organized but I'd think the address could be a dataverse.owner.address or something like that (dataverse.organization.address?)

Reply:
I'm asking the community what they think: https://groups.google.com/g/dataverse-community/c/E3Vfrs757cU/m/HWNvu86yCAAJ

Reply:
@shlake thanks for the feedback! ❤️ 

Two things:

- I like the way you think because it means what I've already coded up should work for you. 😜 You seem ok with dedicated BagIt settings.
- It's interesting that you put "University of Virginia" under "Organization-Address". Technically [RFC 8493](https://datatracker.ietf.org/doc/html/rfc8493) says it's for the "mailing address of the source organization". The example is comma-separated (`Organization-Address: 1 Main St., Cupertino, California, 11111`) so that's probably what I'll suggest in our documentation. (That way I don't have to complicate our code to support multi-line addresses.) As for addresses changing, you're absolutely right! Please use the field in whatever way you feel is best!

Thanks again!

Reply:
I just documented the new BagIt settings: f0f5ac7

That is to say, I'm going with what @shlake wants and letting these settings have their own life independent of `:InstallationName`, `:SystemEmail`, etc.